code/modules/cargo/bounties/reagent.dm

```diff
@@ -175,7 +175,7 @@
 		/datum/reagent/consumable/frostoil,\
 		/datum/reagent/toxin/slimejelly,\
 		/datum/reagent/teslium/energized_jelly,\
-		/datum/reagent/toxin/skewium,\
+		// /datum/reagent/toxin/skewium,\ austation -- removes skewium from possible bounties
```

Comment:
`//` at the start of the line, rather than with tab before it, please. make sure the code itself lines up with the rest for neatness

Reply:
aye